ext/node/polyfills/worker_threads.ts

```diff
@@ -175,17 +175,17 @@ class NodeWorker extends EventEmitter {
         // empty catch block when package json might not be present
       }
       if (
-        !(StringPrototypeEndsWith(
+        (StringPrototypeEndsWith(
           StringPrototypeToString(specifier),
           ".mjs",
         )) ||
         (pkg && pkg.exists && pkg.typ == "module")
```

Comment:
Probably should ensure it's not a `.cjs` file.

Reply:
it's now added!